ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -3405,7 +3405,7 @@ private void GenerateWorkbookStylesPartContent(WorkbookStylesPart workbookStyles
                 foreach (CellFormat f in workbookStylesPart.Stylesheet.CellFormats)
                 {
                     styleId++;
-                    if (CellFormatsAreEqual(f, ss.Value))
+                    if (CellFormatsAreEqual(f, ss.Value, true))
```

Comment:
For boolean parameters, I like naming the parameter. It improves readability.

So `if (CellFormatsAreEqual(f, ss.Value, compareAlignment: true))` , please? Sorry to be pedantic.

Reply:
Agree

Reply:
I have amended the commit